guide/onboarding/automatic-cloud-backup.md

```diff
@@ -0,0 +1,98 @@
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+     image = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-provider.png"
+   retina = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-provider@2x.png"
+   alt-text = "Screen of placeholder where cloud provider UI would be"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-confirmation.png"
+   retina = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-confirmation@2x.png"
+   alt-text = "Screen showing completing a cloud backup"
+   width = 250
+   height = 541
+%}
+
+</div>
+
+### How to Discuss/Explain It
+
+One approach is to explicitly explain within onboarding carousels that a user’s recovery phrase is stored on the cloud; this means that, if they lose or break their device, they can regain access to their funds by logging into their respective cloud account on a different device.
+
+On a similar note, it’s crucial that you also explain that if someone else knows their cloud account information, they can gain access to their funds, too.
+
+{% include tip/open.html color="green" icon="check" label="Do: Inform your Users" %}
+
+Before automatically backing up a recovery phrase via iCloud, make sure the user knows what they are about to do. Once the backup is complete, provide a summary and some reassurances before moving on. This is a good place for loading interactions and animation.
+
+{% include tip/close.html %}
+
+### Tradeoffs
+
+By not making users manually backup, we decrease the risk of self-inflicted loss of funds. Automatic cloud backups put less pressure and responsibility on the user.
+
+While this is an easier way for users to set up a wallet, it brings in the potential threat of malicious third party access. If a user’s cloud account is hacked or they forget their cloud login information, their funds could be compromised.
+
+Additionally, by bringing cloud storage providers into the equation, we assume that users trust their providers. This also raises the question of [decentralization]({{ 'guide/foundations/principles/#decentralization' | relative_url }}): a guiding principle of the bitcoin space and could be considered at conflict with the popular saying, “not your keys, not your coins”.
+
+{% include tip/open.html color="red" icon="forbid" label="Don’t: Use for High Value Amounts" %}
+
+Automatic cloud backups should not be considered for products with users that are storing high-value amounts. This is because there are potentially more security threat vectors. However, it’s important to note that new users will probably not be storing high values in their wallet, making it an appropriate mechanism to implement in a bitcoin product for beginners.
```

Comment:
I don't agree with the assumption that new users will not be storing high value amounts. I think this does occur, new users can still own a considerable amount of bitcoin, especially older generations who have the capital to invest but may not be tech savvy enough to look into proper storage. 

To remedy this I think you should mention that it should be communicated to users using cloud backups that it is not recommended to use this backup method for high value amounts.